developing/dev-setup.mdx

```diff
@@ -59,7 +76,39 @@ Arroyo is built via Cargo, the Rust build tool. To build all of the services in
 ```
 $ cargo build --release
 ```
+Most of the services are pretty light-weight, so are fine to run in debug mode.
```

Comment:
I would specify that this is the control plane services (arroyo-api and arroyo-controller)